src/CredentialsLoader.php

```diff
@@ -28,7 +28,7 @@
  */
 abstract class CredentialsLoader implements FetchAuthTokenInterface
 {
-  const TOKEN_CREDENTIAL_URI = 'https://www.googleapis.com/oauth2/v3/token';
+  const TOKEN_CREDENTIAL_URI = 'https://www.googleapis.com/oauth2/v4/token';
```

Comment:
All our docs now use v4 as the token URL:
https://developers.google.com/identity/protocols/OAuth2InstalledApp

but unsure what the difference is, so was updating this to reflect our docs.